### PR TITLE
Update es_ar.json, es_es.json, es_mx.json

### PR DESCRIPTION
**Pull request by feliclaria**

### CODE REVIEW

File: src/main/resources/assets/endergetic/lang/es_ar.json [hunk 1]
```diff
@@ -72,7 +72,13 @@
 	"block.endergetic.mystical_obsidian_activation_rune": "Runa de activación de obsidiana mística",
 	"block.endergetic.activated_mystical_obsidian_activation_rune": "Runa de activación de obsidiana mística activa",
 	"block.endergetic.acidian_lantern": "Linterna acidiana",
-	"block.endergetic.crystal_holder": "Soporte de cristal",
+	"block.endergetic.crystal_holder": "Soporte de cristales",
+
+	"block.endergetic.ender_campfire": "Fogata de ender",
+  "block.endergetic.ender_torch": "Antorcha de ender",
```
Comment (SmellyModder) [inline src/main/resources/assets/endergetic/lang/es_ar.json:78]: Incorrectly formatted (continues down a bit), should be aligned with the other lines.

File: src/main/resources/assets/endergetic/lang/es_es.json [hunk 2]
```diff
@@ -72,7 +72,13 @@
 	"block.endergetic.mystical_obsidian_activation_rune": "Runa de activación de obsidiana mística",
 	"block.endergetic.activated_mystical_obsidian_activation_rune": "Runa de activación de obsidiana mística activa",
 	"block.endergetic.acidian_lantern": "Linterna acidiana",
-	"block.endergetic.crystal_holder": "Soporte de cristal",
+	"block.endergetic.crystal_holder": "Soporte de cristales",
+
+	"block.endergetic.ender_campfire": "Hoguera de ender",
+  "block.endergetic.ender_torch": "Antorcha de ender",
```
Comment (SmellyModder) [inline src/main/resources/assets/endergetic/lang/es_es.json:78]: Incorrectly formatted (continues down a bit), should be aligned with the other lines.

File: src/main/resources/assets/endergetic/lang/es_mx.json [hunk 3]
```diff
@@ -72,7 +72,13 @@
 	"block.endergetic.mystical_obsidian_activation_rune": "Runa de activación de obsidiana mística",
 	"block.endergetic.activated_mystical_obsidian_activation_rune": "Runa de activación de obsidiana mística activa",
 	"block.endergetic.acidian_lantern": "Linterna acidiana",
-	"block.endergetic.crystal_holder": "Soporte de cristal",
+	"block.endergetic.crystal_holder": "Soporte de cristales",
+
+	"block.endergetic.ender_campfire": "Fogata de ender",
+  "block.endergetic.ender_torch": "Antorcha de ender",
```
Comment (SmellyModder) [inline src/main/resources/assets/endergetic/lang/es_mx.json:78]: Incorrectly formatted (continues down a bit), should be aligned with the other lines.

**Review by SmellyModder — CHANGES_REQUESTED** [review]
Looks good, just some format errors.

**Comment by feliclaria** [comment]
Fixed format

**Review by SmellyModder — APPROVED** [review]

**Pull request merged**